server/utils/routes.js

```diff
@@ -2,18 +2,40 @@ module.exports = {
     getPathFor
 };
 
```

Comment:
This page seems like it is upside down to me. Would be easier to read starting with getPathFor and then reading downwards